lib/src/playable.dart

```diff
@@ -105,13 +107,15 @@ class Metas {
       title.hashCode ^ artist.hashCode ^ album.hashCode ^ image.hashCode;
 
   Metas copyWith({
+    String id,
     String title,
     String artist,
     String album,
     Map<String, dynamic> extra,
     MetasImage image,
   }) {
     return new Metas(
+      id: id ?? this.id,
```

Comment:
used that id so that user can pass their required id than when receiving click notices they can take specific task according to that track. I m using this feature in my own project also and i needed my own track id not auto generated to take specific action based on that passed track id.